masschallenge/django-accelerator
# [AC-9123] Remove Finalist tag from all staff accounts

Changes introduced in [AC-9123](https://masschallenge.atlassian.net/browse/AC-9123)

Remove Finalist tag from all staff accounts

**How to test:**

- Check out this branch
- Run Migrations on your localhost
- Notice that no staff user has a finalist role grant

[you can find earlier reviews here](https://github.com/masschallenge/django-accelerator/pull/447)
